xskillscore/tests/test_contingency.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import numpy.testing as npt
+import pytest
+import xarray as xr
+from sklearn.metrics import confusion_matrix
+
+from xskillscore import Contingency
+
+DIMS = (['time'], ['lon'], ['lat'], 'time', ['lon', 'lat', 'time'])
+CATEGORY_EDGES = [
+    np.array([-np.inf, 0.5, np.inf]),
+    np.array([-np.inf, 0.5, 1.5, np.inf]),
+]
+
+
+@pytest.fixture
+def forecast():
+    times = xr.cftime_range(start='2000', freq='D', periods=10)
+    lats = np.arange(4)
+    lons = np.arange(5)
+    data = np.random.randint(0, 10, size=(len(times), len(lats), len(lons)))
+    return xr.DataArray(data, coords=[times, lats, lons], dims=['time', 'lat', 'lon'])
+
+
+@pytest.fixture
+def observation(forecast):
+    b = forecast.copy()
+    b.values = np.random.randint(0, 10, size=(b.shape[0], b.shape[1], b.shape[2]))
+    return b
+
+
+@pytest.fixture
+def dichotomous_Contingency():
+    observations = xr.DataArray(
+        np.array(2 * [0] + 2 * [1] + 1 * [0] + 2 * [1]), coords=[('x', np.arange(7))]
+    )
+    forecasts = xr.DataArray(
+        np.array(2 * [0] + 2 * [0] + 1 * [1] + 2 * [1]), coords=[('x', np.arange(7))]
+    )
+    category_edges = np.array([-np.inf, 0.5, np.inf])
+    return Contingency(
+        observations, forecasts, category_edges, category_edges, dim=['x']
+    )
+
+
+@pytest.mark.parametrize('type', ['da', 'ds', 'chunked_da', 'chunked_ds'])
+@pytest.mark.parametrize('dim', DIMS)
+@pytest.mark.parametrize('category_edges', CATEGORY_EDGES)
+def test_Contingency_table_values(observation, forecast, category_edges, dim, type):
```

Comment:
It's generally good to include a brief docstring explaining the purpose of the test function. Once you build a large test suite it gets really confusing remembering what each test was for.